static/js/components/inputs/SelectField.js

```diff
@@ -12,8 +13,11 @@ import type {
   UpdateProfileFunc,
 } from '../../flow/profileTypes';
 
+const CREATE_OPTION_REGEX = /^Create option \"|\"$/g;
```

Comment:
hrm if we do a capture group we lose the ability to call `.replace` regardless of what the input is, since `/NO_MATCHES/.exec('a different string') === null`.